<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShedPage</class>
 <widget class="QWidget" name="ShedPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ShedPage{background-color: rgba(113, 37, 37, 1);}</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>49</y>
     <width>430</width>
     <height>370</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>430</width>
     <height>370</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget {
            background-color: white;
            border: 1px solid #d0d0d0;
            border-radius: 10px;
            font-size: 12px;
        }

        QCalendarWidget QWidget {
            alternate-background-color: white;
        }

        /* НАВИГАЦИОННАЯ ПАНЕЛЬ */
        QCalendarWidget QWidget#qt_calendar_navigationbar {
            background-color: #f8f8f8;
            min-height: 35px;
            border-top-left-radius: 10px;
            border-top-right-radius: 10px;
        }

        /* ДНИ НЕДЕЛИ - ЭТО РАБОТАЕТ */
        QCalendarWidget QHeaderView::section {
            background-color: #2196F3;
            color: white;
            font-weight: bold;
            border: none;
            padding: 8px 4px;
            margin: 0px;
        }

        /* ВЫБРАННАЯ ДАТА */
        QCalendarWidget QTableView::item:selected {
            background-color: #2196F3;
            color: white;
            border-radius: 5px;
        }

        /* ОБЫЧНЫЕ ДНИ */
        QCalendarWidget QTableView::item {
            border: none;
            padding: 5px;
        }

        /* КНОПКИ */
        QCalendarWidget QToolButton {
            background-color: transparent;
            color: #333333;
            font-weight: bold;
            border-radius: 5px;
            padding: 5px;
            margin: 2px;
        }

        QCalendarWidget QToolButton:hover {
            background-color: #e0e0e0;
        }</string>
   </property>
   <property name="firstDayOfWeek">
    <enum>Qt::Monday</enum>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="form_container">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>49</y>
     <width>280</width>
     <height>540</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>540</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
                 background-color: rgba(64, 20, 20, 1);
                border-radius: 15px;
                padding: 10px;
            }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>231</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <item>
        <widget class="QPlainTextEdit" name="title_input">
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 5px; border-radius: 15px; background-color: rgba(85, 29, 29, 1);</string>
         </property>
         <property name="placeholderText">
          <string>Название занятия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="time_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>52</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>253</red>
               <green>239</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="204">
               <red>64</red>
               <green>20</green>
               <blue>20</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Martian Mono Condensed</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
        font-family: 'Martian Mono Condensed';
        font-size: 11px;
        padding: 5px;
        border: 1px solid rgba(113, 37, 37, 1);
        border-radius: 8px;
		text: 'egewgew';
        background-color: rgba(64, 20, 20, 0.8);
        color: #fdefc8;
    }
    QLineEdit:focus {
        border: 1px solid rgba(253, 239, 200, 0.5);
		inputMask: '__:__';
	}</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>Время</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>250</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>52</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>253</red>
             <green>239</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>113</red>
             <green>37</green>
             <blue>37</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Martian Mono Condensed</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-family: &quot;Martian Mono Condensed&quot;;
    font-size: 20px;
	color:#fdefc8;
	border-radius: 15px;
	background-color: rgba(113, 37, 37, 1);
}
        
QPushButton:hover {
        background-color: rgba(120, 41, 41, 1);
 }
QPushButton:pressed {
        background-color: rgba(85, 29, 29, 1);
    }</string>
       </property>
       <property name="text">
        <string>Добавить занятие</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
